cypress/page/login.page.ts

```diff
@@ -0,0 +1,34 @@
+class LoginPage {
+    private addemail: string;
+    private addpassword: string;
+    private clickLogin: string;
+    private emailInput: string
+    private passInput: string
+
+    constructor() {
+        this.emailInput = "#email"
+        this.addemail = "aperdomobo@gmail.com";
+        this.passInput = "#passwd"
+        this.addpassword = "WorkshopProtractor";
+        this.clickLogin = "#SubmitLogin";
+
+    }
+
+    public insertMail(): void {
+        cy.get(this.emailInput).type(this.addemail)
+    }
+
+    public insertPassword(): void {
+        cy.get(this.passInput).type(this.addpassword)
```

Comment:
the addpassword property should be an input param of the method

Reply:
Done